### PR TITLE
Add doc for support version range

### PR DESCRIPTION
**Pull request by yaojiefeng**
#8641 is merged. This is the required doc change for it.

### CODE REVIEW

**Comment by awholegunch** [comment]
LGTM. I have edited the original PR. Please check my changes carefully, as I have edited both the descriptions and the syntax example.

**Comment by yaojiefeng** [comment]
Changes LGTM

File: cdap-docs/reference-manual/source/http-restful-api/artifact.rst [hunk 1]
```diff
@@ -501,15 +513,34 @@ an HTTP GET request::
...
+     - Optional plugin scope filter. Scope of the plugin artifact scope. If not specified,
+       defaults to returning plugins in both the ``SYSTEM`` and ``USER`` scopes.
+   * - ``limit``
+     - Optional limit filter. Limits the number of the returned results. If not specified,
+       defaults to returning all plugins which match the requirements.
+   * - ``order``
+     - Optional order filter. Accepted values are one of ``DESC``, ``ASC``, or
+       ``UNORDERED``. ``DESC`` will sort the plugin artifacts first by scope, then by name
+       descending alphabetically, and then by version from highest to lowest; for ``ASC``,
+       sorted first by scope, then by name ascending alphabetically, and then by version
```
Comment (albertshau) [inline cdap-docs/reference-manual/source/http-restful-api/artifact.rst]: should mention that user scope is greater than system scope.

**Review by albertshau — COMMENTED** [review]
just one comment on scope ordering

File: cdap-docs/reference-manual/source/http-restful-api/artifact.rst [hunk 2]
```diff
@@ -537,7 +537,7 @@ an HTTP GET request::
        defaults to returning all plugins which match the requirements.
    * - ``order``
      - Optional order filter. Accepted values are one of ``DESC``, ``ASC``, or
-       ``UNORDERED``. ``DESC`` will sort the plugin artifacts first by scope, then by name
+       ``UNORDERED``. ``DESC`` will sort the plugin artifacts first by scope(``USER`` scope is greater than ``SYSTEM``), then by name
```
Comment (albertshau) [inline cdap-docs/reference-manual/source/http-restful-api/artifact.rst]: insert a space between scope and (

**Review by albertshau — COMMENTED** [review]
minor comment

**Comment by awholegunch** [comment]
I've edited those last few changes. LGTM. Please take a look.

**Review by albertshau — APPROVED** [review]
lgtm

**Pull request merged**